packages/vue/src/components/authenticator.vue

```diff
@@ -452,7 +453,7 @@ const onConfirmVerifyUserSubmitI = (e: Event) => {
  * Update service facade when context updates
  */
 
-const user = ref(null);
+const user: Ref<CognitoUserAmplify | null> = ref(null);
```

Comment:
Vue was having type error `type 'CognitoUserAmplify' is not assignable to type 'null'`, so I added a type assertion here.

Reply:
👍

Reply:
I've seen people also do `ref(null as CognitoUserAmplify)` too, but that just looks funny to me